modules/database/src/main/java/org/eclipse/xpanse/modules/database/serviceconfiguration/update/ServiceConfigurationUpdateRepository.java

```diff
@@ -6,15 +6,32 @@
 
 package org.eclipse.xpanse.modules.database.serviceconfiguration.update;
 
+import jakarta.persistence.LockModeType;
+import jakarta.persistence.QueryHint;
+import java.util.List;
 import java.util.UUID;
 import org.eclipse.xpanse.modules.database.CustomJpaRepository;
+import org.springframework.data.jpa.domain.Specification;
 import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+import org.springframework.data.jpa.repository.Lock;
+import org.springframework.data.jpa.repository.QueryHints;
+import org.springframework.lang.Nullable;
+import org.springframework.stereotype.Repository;
 
 /**
  * Interface to access default JPA methods.
  */
-public interface ServiceConfigurationUpdateRepository extends
-        CustomJpaRepository<ServiceConfigurationUpdateRequest, UUID>,
+@Repository
+public interface ServiceConfigurationUpdateRepository
+        extends CustomJpaRepository<ServiceConfigurationUpdateRequest, UUID>,
         JpaSpecificationExecutor<ServiceConfigurationUpdateRequest> {
 
+    /**
+     * it is possible to read this config from spring properties.
```

Comment:
sorry my mistake. It must be 'It is not possible ..'
Also better to add this comment above line 33